### ISSUE TITLE
cannot create instance

### ISSUE THREAD

**Issue by PSLLSP** [play-with-docker/play-with-docker#340]
I am new top PWK. I logged in and opened my first session. I clicked a button to create new instance. There was a message "creating instance" for few seconds. After short time, message was hidden but I cannot see any new instance, no error or other message. I can press button to create new instance and repeat a try to start new instance...

What is the problem? Is your lab cluster busy and out of free space to launch another instance? Or is there other problem?  Please, report to user why instance was not created.

**Comment by marcosnils** [comment]
try again, it should work better now.

**Comment by PSLLSP** [comment]
It is not better. I  press "CREATE NEW INSTANCE", text changes to "CREATING..." but after two seconds it is replaced with "CREATE NEW INSTANCE". I cannot see any instance. I don't see any error.

**Comment by marcosnils** [comment]
try closing your session and creating a new one.

**Comment by PSLLSP** [comment]
I tried. I use Chromium browser running at Ubuntu 18.04. I tried with Firefox at Ubuntu. No difference. Well, there is a difference, now I see that "creating..." state is shorter, just about 1 second. It was 2 seconds before and, maybe, 5 seconds when I tried my first try.

Just to be sure, this is my procedure. I go to https://labs.play-with-k8s.com/, I login with my GITHUB account, than I start with START button. Than new session with timeout 4 hour is started, I see countdown timer counting down and I can press button "CREATE NEW INSTANCE".
